src/components/VHeader/VFilterButton.vue

```diff
@@ -97,18 +95,15 @@ const VFilterButton = defineComponent({
         return false
       }
       // Below the medium viewport, only show when there's no filters applied.
-      if (!isMinScreenMD.value) {
+      if (!isMinScreenMd.value) {
         return !isHeaderScrolled.value || !filtersAreApplied.value
       }
       return true
     })
 
     // Hide the label entirely when no filters are applied on mobile.
     const showLabel = computed(() => {
-      if (!isMinScreenMD.value && !filtersAreApplied.value) {
-        return false
-      }
-      return true
+      return !(!isMinScreenMd.value && !filtersAreApplied.value)
```

Comment:
thanks for the nice refactor 😄

Reply:
It wasn't me, it was Webstorm's nagging 😆